### DISCUSSION TITLE
Seriennummer mit Buchstaben bei der Hoymiles HMS 1600 4T wird nicht angenommen und somit

### DISCUSSION THREAD

**Discussion by TomekCGN** [tbnobody/OpenDTU#1922] [Q&A]
Hallo Freunde der Sonne,
versuche meine HMS 1600 4T bei der Open DTU anzumelden.
Problemchen ist, dass in dem Feld Einrichtung => Wechelrichter keine Buchstaben bei der Eingabe der Seriennummer angenommen werden und meine HM hat eine Seriennummer die mit 1164A... beginnt und auch mit Buchstaben endet.
Somit kann die die beiden nicht verbinden.
Die Umrechnung in Hexadezimalzahl hat nicht funktioniert.
Ideen?
merci / Danke

**Comment by tbnobody** [comment]
Letzte Version installieren, den Browser Cache löschen oder das webinterface mit Strg+F5 neu laden

**Reply by TomekCGN** [reply]
Besten Dank. Das habe ich mir beinahe gedacht... aber was tun wenn das update nicht klappt?
![A2](https://github.com/tbnobody/OpenDTU/assets/167441950/18929b4f-4384-4c43-9e30-e881372da147)
![a1](https://github.com/tbnobody/OpenDTU/assets/167441950/a0568821-8f35-4141-8649-494d8ba8578c)

**Reply by TomekCGN** [reply]
Habe es mit der letzten, Vorletzten und der v23.4.5 probiert. Weiter kann ich es im Archiv nicht einsehen.
opendtu-generic.bin - korrekt?

**Reply by tbnobody** [reply]
Das bedeutet das deine Firmware bzw erstinstallation viel zu alt ist. Vor einem Jahr wurde das partitionsschema geändert. 

https://www.opendtu.solar/firmware/howto/upgrade_partition/

**Reply by cat24max** [reply]
@tbnobody So alt ist das Fusion-Board nicht :)
Wir liefern derzeit mit einer paar Monate alten Firmware aus. Hintergrund ist, dass ich das WLAN-AP-Passwort entfernt habe und den Werksreset ohne Login ermoeglicht habe. Leider vergisst der ESP32 hin und wieder mal das WLAN-Passwort und dann geht gar nichts mehr.
Ich habe @TomekCGN auf seine E-Mail an uns geantwortet.

VG
Tim von AllianceApps

**Comment by TomekCGN** [comment]
Hallo Freunde der Sonne,
vielen Dank für die Rückmeldungen und Hinweise.
Letztendlich hat Mastertape [mastertape](https://github.com/mastertape) gelungen das Problem zu lösen.

Ja - es musste ein Update gemacht werden aber - lt. Tutorials sollte ich mit opendtu-generic updaten. Es funktionierte nicht - wie oben beschrieben. Mastertape hat es dann mit opendtu-generic_esp32s3_usb gemacht.
Somit hatte ich die neuste Ver aber die Seriennummer mit Buchstaben wurde immer noch nicht angenommen. 
Es müsste noch die Json Datei aktualisiert werden: opendtu_fusion.json - so jetzt läufts.
Die Seriennummer mit Buchstaben wird angenommen . ich kann auf meinen WR zugreifen.

![config](https://github.com/tbnobody/OpenDTU/assets/167441950/a60fa40b-6efa-483b-b0df-0e72080746d5)
![config2](https://github.com/tbnobody/OpenDTU/assets/167441950/3fae1457-8c93-4cb4-a0c1-bcecce35e4cb)
